content/docs/sidebar.yaml

```diff
@@ -1,29 +1,50 @@
 - title: Overview
   items:
-    - title: Introduction
+    - title: What is Neon
       slug: cloud/about
-    - title: Getting Started
+    - title: Getting Started with Neon
       slug: cloud/getting-started
-    - title: Concepts
-      slug: cloud/concepts
-    - title: Tutorials
-      slug: cloud/tutorials
-- title: Quickstart
+    - title: Signing Up
+      slug: get-started-with-neon/signing-up
+    - title: Setting up a Project
+      slug: get-started-with-neon/setting-up-a-project
+    - title: Using API Keys
+      slug: get-started-with-neon/using-api-keys
+    - title: Query with Neon's SQL Editor
+      slug: get-started-with-neon/query-with-neon-sql-editor
+    - title: Query with Neon's psql Editor
+      slug: get-started-with-neon/query-with-psql-editor
+- title: Integrations
   items:
-    - title: Signing up
-      slug: quickstart/console
-    - title: Run a Postgres app
-      slug: quickstart/postgres
+    - title: Cloud Integrations
+      slug: integrations/integrations-list
     - title: Run a Hasura app
       slug: quickstart/hasura
+    - title: Run a Vercel and Next.js app
+      slug: quickstart/vercel
     - title: Run a Prisma app
       slug: quickstart/prisma
+    - title: Frameworks & Programming Languages
+      slug: integrations/integrations-list
+    - title: Run a Go app
+      slug: quickstart/go
+    - title: Run a Java app
+      slug: quickstart/java
     - title: Run a Django app
       slug: quickstart/django
-    - title: Run a SQL Alchemy app
-      slug: quickstart/sqlalchemy
     - title: Run a Node.js app
       slug: quickstart/node
+<<<<<<< HEAD
```

Comment:
conflict needs to be resolved before the merge

Reply:
I am not sure why the files in quickstart are having conflicts since they remain in the same file structure @stepashka